manifests/repo/download_postgresql_org.pp

```diff
@@ -17,14 +17,14 @@
     $label2 = 'rhel'
   }
 
-  yumrepo { 'yum.postgresql.org':
```

Comment:
@DavidS +1 to both suggestions, setting a repo name thats less prone to change like 'postgresql.org' is a good idea, while we ensure the old is purged also ...